### PR TITLE
README - note for protobuf-maven-plugin in Eclipse

### PR DESCRIPTION
**Pull request by paulvi**
ref https://github.com/xolstice/protobuf-maven-plugin/issues/16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,8 @@ For protobuf-based codegen integrated with the Maven build system, you can use
 </build>
 ```
 
+For Eclipse IDE check https://github.com/trustin/os-maven-plugin#issues-with-eclipse-m2e-or-other-ides
```
Comment (carl-mastrangelo) [inline README.md]: Could you put this in markdown syntax like the link below

**Comment by carl-mastrangelo** [comment]
@ejona86 FYI

File: README.md (hunk 1 above)
Comment (paulvi) [inline README.md]: This is so tiny update, can you mention the link in any style,
as you can without new commit/PR

ref xolstice/protobuf-maven-plugin#16

**Comment by ejona86** [comment]
@paulvi, thank you for tracking down the issue with Eclipse/NetBeans. I'm quite disappointed (in the ecosystem fragmentation) that it was that hard. I made #2420.

**Comment by ejona86** [comment]
#2420 is now in.

**Pull request closed without merging**